helm/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for space-cloud.
+
+namespace: "space-cloud" # K8s namespace to deploy space cloud components
```

Comment:
We shouldn't keep this configurable. SC has a hard dependency on the namespace right now